File: WebApi/Models/CardModel.cs
using System;
using System.Collections.Generic;

namespace WebApi.Models
{
    public class CardModel
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public double CreateDateTime { get; set; }
        public double? DeadLineDateTime { get; set; }
        public double? StartPanicDateTime { get; set; }
        public double? PanicIntervalInMiliseconds { get; set; }
        public int DefferalCount { get; set; }
        public int Status { get; set; }
        public int Priority { get; set; }
        public List<CardCommentModel> CardComments { get; set; }
        public CardContentModel Content { get; set; }
    }
}

File: AngularClient/src/app/modules/content/models/card.model.ts
import { CardContentModel } from './caed-content.model';
import { CardCommentModel } from './card-comment.model';
import { ICardModel } from '../@types/tasks';

export class CardModel{
    id: string;
    name: string;
    createDateTime: Date;
    deadLineDateTime : Date;
    startPanicDateTime : Date;
    panicIntervalInMiliseconds: number;
    defferalCount: number;
    status: number;
    priority: number;
    content: CardContentModel;
    cardComments: CardCommentModel[];

    constructor(data: ICardModel){
        this.id = data.id;
        this.name = data.name;
        this.createDateTime = new Date(data.createDateTime);
        this.deadLineDateTime = data.deadLineDateTime != null ? new Date(data.deadLineDateTime) : null;
        this.startPanicDateTime = data.startPanicDateTime != null ? new Date(data.startPanicDateTime) : null;
        this.panicIntervalInMiliseconds = data.panicIntervalInMiliseconds;
        this.defferalCount = data.defferalCount;
        this.status = data.status;
        this.priority = data.priority;
        this.content = data.content!= null ? new CardContentModel(data.content) : null;
        this.cardComments = data.cardComments != null ?  data.cardComments.map(x=> new CardCommentModel(x)) : [];
    }
}